### PR TITLE
Add wrapper for GraPhlAn

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

An other tool migration :)

Bérénice

### CODE REVIEW

File: tools/graphlan/graphlan.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <inputs>
+        <param name="input_tree" type="data" format="txt,nhx,phyloxml" label="Input tree"/>
+        <param argument='--format' type="select" label="Output format">
+            <option value="png" selected="true">PNG</option>
+            <option value="pdf">PDF</option>
+            <option value="ps">PS</option>
+            <option value="eps">EPS</option>
+            <option value="svg">SVG</option>
+        </param>
+        <param argument="--dpi" type="integer" label="Dpi of the output image" help="For non vectorial formats" optional="True"/>
```
Comment (bgruening) [inline tools/graphlan/graphlan.xml]: This could be in an conditional, isn't it?

File: tools/graphlan/graphlan.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    </tests>
+    <help><![CDATA[
+**What it does**
+
+GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.
+
+For more information, check the `user manual <https://bitbucket.org/nsegata/graphlan/overview>`_.
+
+    ]]></help>
+    <citations>
```
Comment (bgruening) [inline tools/graphlan/graphlan.xml]: This does not have a citation?

Comment (nsoranzo) [inline tools/graphlan/graphlan.xml]: 10.7717/peerj.1029

Comment (bgruening) [inline tools/graphlan/graphlan.xml]: Thanks Nicola, I guess this should then end up in a macro as well.

File: tools/graphlan/graphlan.xml (hunk 1 above)
Comment (bebatut) [inline tools/graphlan/graphlan.xml]: Done

File: tools/graphlan/graphlan.xml (hunk 2 above)
Comment (bebatut) [inline tools/graphlan/graphlan.xml]: Added. Thanks Nicola

**Comment by bgruening** [comment]
Thanks @bebatut and  @nsoranzo!

**Pull request merged**